backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java

```diff
@@ -32,7 +32,9 @@
   private UACServiceGrpc.UACServiceBlockingStub uacServiceBlockingStub;
   private TeamServiceGrpc.TeamServiceBlockingStub teamServiceBlockingStub;
   private OrganizationServiceGrpc.OrganizationServiceBlockingStub organizationServiceBlockingStub;
-  public static boolean isMigrationUtilsCall = false;
+  public static boolean isBackgroundUtilsCall = false;
```

Comment:
What is this? It seems like it's a single global variable shared by everything using the auth service for calls, which is a **big** no-go. We should not use the service account permissions for non-service account requests.